frontend/src/constants/states.ts

```diff
@@ -36,38 +38,36 @@ export const STATES = [
   { shortName: 'ny', longName: 'New York' },
   { shortName: 'nc', longName: 'North Carolina' },
   { shortName: 'nd', longName: 'North Dakota' },
+  { shortName: 'mp', longName: 'Northern Mariana Islands' },
   { shortName: 'nv', longName: 'Nevada' },
   { shortName: 'oh', longName: 'Ohio' },
   { shortName: 'ok', longName: 'Oklahoma' },
   { shortName: 'or', longName: 'Oregon' },
   { shortName: 'pa', longName: 'Pennsylvania' },
+  { shortName: 'pr', longName: 'Puerto Rico' },
   { shortName: 'ri', longName: 'Rhode Island' },
   { shortName: 'sc', longName: 'South Carolina' },
   { shortName: 'sd', longName: 'South Dakota' },
   { shortName: 'tn', longName: 'Tennessee' },
   { shortName: 'tx', longName: 'Texas' },
   { shortName: 'ut', longName: 'Utah' },
   { shortName: 'vt', longName: 'Vermont' },
+  { shortName: 'vi', longName: 'Virgin Islands' },
   { shortName: 'va', longName: 'Virginia' },
   { shortName: 'wa', longName: 'Washington' },
   { shortName: 'wv', longName: 'West Virginia' },
   { shortName: 'wi', longName: 'Wisconsin' },
-  { shortName: 'wy', longName: 'Wyoming' },
-  { shortName: 'as', longName: 'American Samoa' },
-  { shortName: 'gu', longName: 'Guam' },
-  { shortName: 'mp', longName: 'Northern Mariana Islands' },
-  { shortName: 'pr', longName: 'Puerto Rico' },
-  { shortName: 'vi', longName: 'Virgin Islands' }
+  { shortName: 'wy', longName: 'Wyoming' }
 ]
 
 /**
  * TODO: Codegen
  */
-export type State = 'al' |'ak' |'ar' |'az' |'ca' |'co' |'ct' |'de' |'dc' |'fl' |'ga' |'hi' |'id' |'il' |'in' |'ia' |'ks' |'ky' |'la' |'me' |'md' |'ma' |'mi' |'mn' |'mo' |'ms' |'mt' |'ne' |'nh' |'nj' |'nm' |'ny' |'nc' |'nd' |'nv' |'oh' |'ok' |'or' |'pa' |'ri' |'sc' |'sd' |'tn' |'tx' |'ut' |'vt' |'va' |'wa' |'wv' |'wi' |'wy'
-| 'as' | 'gu' | 'mp' | 'pr' | 'vi'
+export type State = 'al' | 'ak' | 'ar' | 'az' | 'ca' | 'co' | 'ct' | 'de' | 'dc' | 'fl' | 'ga' | 'hi' | 'id' | 'il' | 'in' | 'ia' | 'ks' | 'ky' | 'la' | 'me' | 'md' | 'ma' | 'mi' | 'mn' | 'mo' | 'ms' | 'mt' | 'ne' | 'nh' | 'nj' | 'nm' | 'ny' | 'nc' | 'nd' | 'nv' | 'oh' | 'ok' | 'or' | 'pa' | 'ri' | 'sc' | 'sd' | 'tn' | 'tx' | 'ut' | 'vt' | 'va' | 'wa' | 'wv' | 'wi' | 'wy'
+  | 'as' | 'gu' | 'mp' | 'pr' | 'vi'
 
 export const CaliforniaFKMServiceAreas = [
```

Comment:
why is this still here?

Reply:
+1

Reply:
Remember when the datasets used to live in `states.ts`? Good times.

Reply:
Yeah, can we get rid of this?